### DISCUSSION TITLE
Please stop locking discussions that are not stale or old

### DISCUSSION THREAD

**Discussion by coreyward** [typescript-eslint/typescript-eslint#10105] [Community Feedback]
There are a dozen threads on a topic I am also having issues with w/r/t version 8, but despite only being a few months old, all of them are locked. I don't have enough to say to open a new discussion and re-introduce the paragraphs and paragraphs of context, but I do have a bit of context to contribute, but you have a Github action that locks discussions after just 7 days. I suspect you're shooting yourself in the foot with this, because it means the context for each topic is spread across a dozen different threads. It also makes it hard for you to see that you don't seem to understand _how broad an impact_ a particular issue has by virtue of preventing even voting. 

So please, stop locking discussions so quickly and shutting the community out.

Edit: Also ironic that users have to check a box that says they've searched for a relevant discussion before they open another one when you didn't let them participate in the original discussion.

**Comment by bradzacher** [comment]
We lock them on purpose.
People would often comment on old threads and just say "I still have this issue". This comment is frustrating and useless for us as maintainers because it provides zero context for us to investigate - so then we need to have lengthy back-and-forth with someone via comments to get a reproduction.
Often we would also go through all that work only to find that the person was just on an old version before the fix.

So we lock closed issues to raise the bar. This forces people to have to file a new issue and create a reproduction. It means that all those people on old versions now figure that out on their own whilst creating a reproduction. It means that if there is an issue that we don't need to waste time going back and forth asking for a repro.

You're not wrong that raising the bar like this also means that people might just choose to not engage at all. But realistically we are *_volunteer_* maintainers with very limited bandwidth. Saving bandwidth and reducing noise is arguably more important for us so that we can more effectively spend our time building things and investigating bugs with repros.

> I don't have enough to say to open a new discussion and re-introduce the paragraphs and paragraphs of context, but I do have a bit of context to contribute

That's fine - it's perfectly acceptable to file a new issue and reference the old one. You just need to fill in all the fields and provide your new repro as well.

People do this all the time and we welcome such contributions.

> Also ironic that users have to check a box that says they've searched for a relevant discussion before they open another one 

This is to encourage people to search for answers before posting a new issue. We have FIVE THOUSAND issues - there's often already the answer someone is looking for.

This also encourages people to find open issues. Esp when there's a bug - it's not uncommon that someone will try to report it twice - but nudging people to search can help stop someone from creating a new duplicate issue and instead reacting to an existing open one.

**Reply by bradzacher** [reply]
Also worth noting that we have a discord that h the people can come to chat with us without the structure and rigor of issues.

**Comment by JoshuaKGoldberg** [comment]
> a dozen threads on a topic I am also having issues with w/r/t version 8

Which threads, specifically? Can you provide specific examples / links please? Your post is very to-the-point (nicely done there) but it's hard to really act on it without specifics.

Also, when you say _"discussion"_, do you mean GitHub Discussions on https://github.com/typescript-eslint/typescript-eslint/discussions? I'm searching on https://github.com/typescript-eslint/typescript-eslint/discussions?discussions_q=is%3Aclosed+is%3Alocked and see very few locked discussions. Most of the discussions there were closed as invalid, such as requests for support (we don't use GH Discussions for that).

> Edit: Also ironic that users have to check a box that says they've searched for a relevant discussion before they open another one when you didn't let them participate in the original discussion.

Nit: this is not irony 😜. Most of the time in our experience, the act of searching for past discussions + reading them gives people the answers they search for.
